docarray/array/storage/qdrant/backend.py

```diff
@@ -37,6 +39,10 @@ class QdrantConfig:
     collection_name: Optional[str] = None
     host: Optional[str] = field(default="localhost")
     port: Optional[int] = field(default=6333)
+    grpc_port: Optional[int] = field(default=6334)
+    prefer_grpc: Optional[bool] = field(default=True)
```

Comment:
follow db default behaviour:
```suggestion
    grpc_port: Optional[int] = None
    prefer_grpc: Optional[bool] = None
```